templates/KpisList/makedata.js

```diff
@@ -0,0 +1,109 @@
+import { HvTooltip, HvTypography } from "@hitachivantara/uikit-react-core";
+import { Level0Good, Level1, Level2Average, Level3Bad } from "@hitachivantara/uikit-react-icons";
+
+const getEntry = (opts) => opts[Math.floor(Math.random() * opts.length)];
+
+function getDate() {
+  const start = new Date(2018, 1, 1);
+  const end = new Date();
+  const date = new Date(start.getTime() + Math.random() * (end.getTime() - start.getTime()));
+
+  return `${date.getFullYear()}/${date.getMonth() + 1}/${date.getDay()}`;
+}
+
+const getRandomStatus = () => {
+  return Math.floor(Math.random() * 4);
+};
+
+const getServerID = () => Math.floor(100000000 + Math.random() * 1000000000);
+
+const getBuild = () => {
+  const characters = "ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789";
+  let result = "";
+  const charactersLength = characters.length;
+  for (let i = 0; i < 6; i += 1) {
+    result += characters.charAt(Math.floor(Math.random() * charactersLength));
+  }
+  return result;
+};
+
+const entries = [
+  { name: "Previous", description: "Clean Data Logs" },
+  { name: "Home", description: "Review Log" },
+  { name: "Carriage", description: "Deploy Cloud Run" },
+  { name: "Black", description: "Clean Session" },
+  { name: "Forward", description: "Update Build" },
+];
+
+const newEntry = (i) => {
+  const entry = getEntry(entries);
+  return {
+    id: `${i + 1}`,
+    name: entry.name,
+    description: entry.description,
+    serverId: getServerID(),
+    created: getDate(),
+    build: getBuild(),
+    status: getRandomStatus(),
+  };
+};
+
+export const makeData = (len = 10) => {
+  const data = [];
+  for (let i = 0; i <= len; i += 1) {
+    data.push(newEntry(i));
+  }
+  return data;
+};
+
+export const getColumns = () => [
+  {
+    Header: "Status",
+    accessor: "status",
+    style: { width: 40 },
+    Cell: (cellData) => {
+      switch (cellData.row.original.status) {
+        case 0:
+          return (
+            <HvTooltip title={<HvTypography>Success</HvTypography>}>
+              <div>
+                <Level0Good semantic="sema1" iconSize="XS" />
+              </div>
+            </HvTooltip>
+          );
+        case 1:
+          return (
+            <HvTooltip title={<HvTypography>Error</HvTypography>}>
+              <div>
+                <Level3Bad semantic="sema4" iconSize="XS" />
+              </div>
+            </HvTooltip>
+          );
+        case 2:
+          return (
+            <HvTooltip title={<HvTypography>Open</HvTypography>}>
+              <div>
+                <Level2Average semantic="sema3" iconSize="XS" />
+              </div>
+            </HvTooltip>
+          );
+        case 3:
+          return (
+            <HvTooltip title={<HvTypography>Unassigned</HvTypography>}>
+              <div>
+                <Level1 semantic="sema2" iconSize="XS" />
+              </div>
+            </HvTooltip>
+          );
+        default:
+          return <></>;
```

Comment:
to avoid this warning maybe we could use one case as a default or alternative create a dummy unknown status component, because otherwise this might break